tests/test_edgeql_introspection.py

```diff
@@ -637,6 +637,29 @@ class TestIntrospection(tb.QueryTestCase):
             ]
         )
 
+    async def test_edgeql_introspection_function_02(self):
+        await self.assert_query_result(
+            r"""
+                WITH MODULE schema
+                SELECT `Function` {
+                    name,
+                    session_only
+                }
+                FILTER .name IN {'std::count', 'sys::advisory_lock'}
+                ORDER BY .name;
+            """,
+            [
+                {
+                    "name": "std::count",
+                    "session_only": {},
```

Comment:
I don't see this as being resolved.  Why is `{}` still here?

Reply:
I made a separate branch with some more schema introspection tests, but then forgot to update this specific one. I've changed it to `False` now and marked it with `xfail`.